### ISSUE TITLE
Using `with-instrument-disabled` generates warnings in CLJS 1.10.439

### ISSUE THREAD

**Issue by bhb** [jeaye/orchestra#30]
Repro:

`clojure -Sdeps '{:deps {orchestra {:mvn/version "2018.11.07-1"} org.clojure/clojurescript {:mvn/version "1.10.439"}}}' --main cljs.main`

```
cljs.user=> (require '[orchestra-cljs.spec.test :as orch.st])
nil
cljs.user=> (orch.st/with-instrument-disabled)
WARNING: var: orchestra-cljs.spec.test/*instrument-enabled* is not public at line 1 <cljs repl>
WARNING: var: orchestra-cljs.spec.test/*instrument-enabled* is not public at line 1 <cljs repl>
nil
```

Actual: warnings
Expected: no warnings 😄

**Comment by jeaye** [comment]
This should be resolved now, in `2018.12.06-2`. Let me know if it's not!

**Comment by bhb** [comment]
Works great. Thanks for fixing this! 💯
